packages/jam/jam-host-calls/accumulate/new.test.ts

```diff
@@ -23,13 +23,16 @@ const CODE_LENGTH_REG = 8;
 const GAS_REG = 9;
 const BALANCE_REG = 10;
 const GRATIS_STORAGE_REG = 11;
+const SERVICE_ID_REG = 12;
 
 function prepareRegsAndMemory(
   codeHash: CodeHash,
   codeLength: U64,
   gas: U64,
   balance: U64,
   gratisStorage: U64,
+  // If value exceeds 2 ** 16, it's just ignored.
+  serviceId: U64 = tryAsU64(2 ** 32 - 1),
```

Comment:
Is this stuff related to the new registrar? I'm a bit confused what exactly is the meaning of this and what's the comment about.

```suggestion
  // If value exceeds 2 ** 16 or the service is not registrar, it's ignored.
  wantedServiceId: U64 = tryAsU64(2 ** 32 - 1),
```

This kind of wording makes it clearer for me, but I'm not sure if that's correct.